ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -55,8 +56,8 @@ oc get csv ${INSTALLED_CSV} -o jsonpath='{.spec.install.spec.deployments[*].name
 timeout ${TIMEOUT_OPERATORS_AVAILABLE} xargs -I {} -d ' ' \
 sh -c 'oc wait --for=condition=Available deployment {} --timeout=-1s'
 
-# Deploy openstack services
-make openstack_deploy
+# Deploy openstack services with the sample from the PR under test
+OPENSTACK_CR=/go/src/github.com/${ORG}/${OPENSTACK_OPERATOR}/samples/core_v1beta1_openstackcontrolplane.yaml make openstack_deploy
```

Comment:
i think that config dir is missing in path:
"config/samples/core_v1beta1_openstackcontrolplane.yaml"

Reply:
Maybe we should try to fetch the entire code user's commit and place in OPERATOR_BASE_DIR.
But this will require more work and tests. We can go with this if works to unblock other PRs

Reply:
> i think that config dir is missing in path: "config/samples/core_v1beta1_openstackcontrolplane.yaml"

You are right. I've fixed that now. 

> Maybe we should try to fetch the entire code user's commit and place in OPERATOR_BASE_DIR.

I assume that `/go/src/github.com/${ORG}/${OPENSTACK_OPERATOR}` is the clone of the repo where the PR is checked out. Isn't it?

Reply:
It is, but I'm considering providing OPENSTACK_REPO and OPENSTACK_BRANCH to install_yamls, so it can fetch the PR code in OPERATOR_BASE_DIR, instead of cloning 'github.com/openstack-k8s-operators/openstack-operator@master'.
I will try that in another PR.

Reply:
hm, yeah, let's do that in a separate PR, but that make sense too.